src/model/resources/vre_stor/vre_stor.jl

```diff
@@ -2034,28 +2035,28 @@ function vre_stor_capres!(EP::Model, inputs::Dict, setup::Dict)
 
     #Variable costs of DC "virtual charging" for technologies "y" during hour "t" in zone "z"
     @expression(EP, eCVar_Charge_DC_virtual[y in DC_CHARGE,t=1:T],
-        inputs["omega"][t]*by_rid(y,:Var_OM_Cost_per_MWh_Charge_DC)*vCAPRES_DC_CHARGE[y,t]/by_rid(y,:EtaInverter))
+        inputs["omega"][t]*(setup["VirtualChargeDischargeCost"]./scale_factor)*vCAPRES_DC_CHARGE[y,t]/by_rid(y,:EtaInverter))
```

Comment:
You could define a new variable and then use it in all the expressions below (in this way, you would divide only once):
```
virtual_chargedischarge_cost = setup["VirtualChargeDischargeCost"] ./ scale_factor
@expression(EP, eCVar_Charge_DC_virtual[y in DC_CHARGE,t=1:T],
        inputs["omega"][t]*(virtual_chargedischarge_cost * vCAPRES_DC_CHARGE[y,t]/by_rid(y,:EtaInverter))
```